readthedocs/templates/errors/404/no_language.html

```diff
@@ -0,0 +1,10 @@
+{% extends "errors/404/base.html" %}
+{% load i18n %}
+
+{# project exists but not in the requested language #}
+
+{% block 404_error_message %}
+  {% include "errors/404/include_error_message.html" with not_found_subject="language" %}
```

Comment:
Totally agree! The `not_found_subject` itself is also not translatable. The approach was nice to avoid duplicating the same text in each template while developing the solution. But ultimately, it's not nice.